components/page-end-cta.jsx

```diff
@@ -0,0 +1,17 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var PageEndCTA = React.createClass({
+  render: function() {
+    return (
+      <div className="page-end-cta">
+        <p>{this.props.description}</p>
```

Comment:
Similar to my [suggestion about `Blockquote`](https://github.com/mozilla/teach.webmaker.org/pull/193/files#r26265692) above, I think that using `this.props.children` instead of `this.props.description` here might be nice, especially if the need for markup in the description ever arises.